docs/DeveloperGuide.md

```diff
@@ -212,6 +212,41 @@ The following activity diagram summarizes what happens when a user executes a ne
 - The Review adding commands are straight-to-the-point and efficient for users to add Review for Stall in FoodWhere.
 - The prefixes allow users to understand what the different types of data fields Review need in order to be created.
 
+### Finding stalls and reviews feature
+
+#### What is finding stalls and reviews feature about?
+
+This feature is used to find stalls and reviews in `AddressBook` by name and/ or by tags. It uses the following two commands:
```

Comment:
There also seems to be a space rather consistently after the slash, I think a later pass is also required to make that consistent later since we are bound to make the mistake of making this inconsistent.

Reply:
Fixed